### PR TITLE
Css fixes 2

### PR DESCRIPTION
**Pull request by lfbergee**
## 📥 Proposed changes

- Buttons had the rather ugly dottet line around on firefox.
- Wierd different color background on select in datepicker

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/datepicker/datepicker.scss [hunk 1]
```diff
@@ -47,10 +47,12 @@ $background-color: $snøhvit;
                 display: block;
                 box-shadow: inset 0 rem(-2px) 0 0 $svart;
                 transition: box-shadow 175ms ease-in;
+                background-color: $snøhvit;
+                text-transform: capitalize;
 
                 font-size: 1.75rem;
                 line-height: 3rem;
-                padding-bottom: $component-spacing--small;
+                height: rem(50px);
```
Comment (piofinn) [inline packages/datepicker/datepicker.scss]: Why 50px? If we're consolidating this with text-input and dropdown it should be 40px (32px line height + 8px bottom padding)

Comment (lfbergee) [reply packages/datepicker/datepicker.scss]: Actually the height of the input field is 48px, so it should be 48, not 50

Comment (lfbergee) [reply packages/datepicker/datepicker.scss]: But that does not look right, 50px does look correct.

Comment (piofinn) [reply packages/datepicker/datepicker.scss]: Huh, OK.. I was looking at the dropdown style, where it is 40px (plus bottom border). We definitely need to have the designers take a look at these components to find common values 😅

**Pull request merged**